site/docs/components/button/accessibility.mdx

```diff
@@ -1,35 +1,45 @@
 ---
-title: Button
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
 layout: DetailComponent
 sidebar:
   exclude: true
 data:
   $ref: ./#/data
 ---
 
-### Best practises
+### Best practices
 
-#### When a Button has both an icon and a label
+- When a Button has both an icon and text, pass `aria-hidden=”true”` to the icon so that its text is not announced by screen readers.
+- When a Button has an icon only and no text, pass an `aria-label` to Button that describes its purpose.
 
-- Pass `aria-hidden=”true”` to the icon so that its label is not announced by screen readers.
+#### Focusable disabled button
 
-#### When a Button has an icon only and no label
+- By default, a disabled button is not focusable. However, in certain scenarios it may be preferable for a disabled button to receive focus. To determine if this is necessary, the following rule of thumb can be followed: "If a disabled element can be enabled (either by the user or by the system) it should be made focusable.” For further information, view the W3C guidance on the [`aria-disabled` attribute](https://www.w3.org/TR/wai-aria/#aria-disabled).
 
-- Pass an `aria-label` to Button that describes its purpose.
+#### Screen reader behavior:
+
+On focus, the screen reader should:
+
+- Identify the button
+- Read out the text,
+- Express the state
+  - In certain situations, the screen reader may need to indicate if the button has a popup for lists or menus.
 
 ### Keyboard interactions
 
 <KeyboardControls>
 <KeyboardControl keyOrCombos="Tab">
 
-- If focus is above the button, Tab key press moves focus onto the button.
-- If the button is disabled and is set to receive focus, receives focus.
-- If the button has focus, Tab moves focus out of the button, to next focusable component in the tab order.
+- Tab moves focus onto the button.
+- If the button is disabled and not set to receive focus, Tab moves focus to the next focusable component.
+- If the button has focus, Tab moves focus to the next focusable component in the tabbing order.
 
 </KeyboardControl>
 <KeyboardControl keyOrCombos="Tab + Shift">
```

Comment:
Why this is "Tab + Shift" but most others are "Shift + Tab"?

Reply:
updated to align with all docs